GoMeet/src/main/java/com/google/sps/data/ErrorMessages.java

```diff
@@ -11,7 +11,7 @@
   public static final String TOO_MANY_RESULTS_ERROR = 
       "Too many results returned";
   public static final String BAD_REQUEST_ERROR_LOCATION = 
```

Comment:
Optional: The name of the error message constant may need to be made more specific if the message is specific.

[Side note: The way that error messages are grouped and named is something I've been thinking would've needed refactoring if we had more time - e.g. whether to have a set of generic error messages that are shared between all the code (fewer error message constants, easier to manage, but less useful for the debugger), or very specific ones that can only be used by one or two functions (may be more difficult to update later if message needs to be changed). Because currently we have a combination of both, but they're all in one shared file which is confusing.]

Reply:
[Side note: The way that error messages are grouped and named is something I've been thinking would've needed refactoring if we had more time - e.g. whether to have a set of generic error messages that are shared between all the code (fewer error message constants, easier to manage, but less useful for the debugger), or very specific ones that can only be used by one or two functions (may be more difficult to update later if message needs to be changed). Because currently we have a combination of both, but they're all in one shared file which is confusing.]

Reply:
Yeah, you're right. In hindsight, some of these messages could be combined. For example, a few of them are used for incorrect request parameters.